gramex/handlers/mlhandler.py

```diff
@@ -148,7 +158,7 @@ def set_opt(cls, key, value):
             transform[key] = value
             cls.config_store.dump('transform', transform)
             cls.config_store.update['transform'] = transform
-        elif key in ('class', 'params'):
+        elif key in ('class', 'params', 'score_pref'):
             model = cls.config_store.load('model', {})
```

Comment:
@prakrutisingh24 All of this code in the setup function is running only once at the beginning, i.e. when the model is being _trained_. Scoring the model doesn't make sense here. Scoring actually happens only later when the user sends a POST request. Therefore, all of our scoring code should happen in that function. We can remove all this code.